eventmesh-common/build.gradle

```diff
@@ -65,9 +61,8 @@ dependencies {
     testImplementation "com.google.guava:guava"
 
     testImplementation "org.slf4j:slf4j-api"
-    testImplementation "org.apache.logging.log4j:log4j-api"
-    testImplementation "org.apache.logging.log4j:log4j-core"
-    testImplementation "org.apache.logging.log4j:log4j-slf4j2-impl"
+    testRuntimeOnly "org.apache.logging.log4j:log4j-core"
+    testRuntimeOnly "org.apache.logging.log4j:log4j-slf4j2-impl"
```

Comment:
EventMesh runs as a server most of the time, and we have a `start.sh` script that starts the server by executing the
`org.apache.eventmesh.starter.StartUp` to start the server. Only artifacts under the `eventmesh-sdk` module and the `eventmesh-common` module will be referenced by third-party applications.

When I run the `StartUp` class, I get an error that I think is related to `testRuntimeOnly`.

![image](https://github.com/apache/eventmesh/assets/41445332/33779986-dd95-4783-971e-d31744a32630)

[EventMesh Runtime Quick Start | Apache EventMesh](https://eventmesh.apache.org/docs/instruction/runtime/)

Reply:
Starting EventMesh with `start.sh` should work the same way, since I added the logging backend in the `dist` task.

Now I also restored it in the `eventmesh-starter` artifact, so it should be fine.